packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -122,6 +130,22 @@ class TestCommand extends FlutterCommand {
       );
     }
 
+    final bool watchTests = argResults['watch'];
+
+    if (watchTests) {
```

Comment:
Since these checks don't need the `files` check, I would move them into validateCommand

Reply:
`validateCommand` doesn't take any arguments, so I am having trouble understanding how I would move this check there?

Reply:
Done.